aldryn_config.py

```diff
@@ -69,6 +69,17 @@ class Form(forms.BaseForm):
             '"aldryn project up" to re-create the db container.'
         )
     )
+    prefix_default_language = forms.CheckboxField(
+        'Remove URL language prefix for default language',
+        required=False,
+        initial=True,
+        help_text=(
+            'For example, http://example.com/ rather than '
+            'http://example.com/en/ if en (English) is the default language.'
+            'If multiple languages are configured, this option will be ignored for '
+            'django versions lower than 1.10'
```

Comment:
`For example, <i>http://example.com/</i> rather than <i>http://example.com/en/</i> if <i>en</i> (English) is the default language.
 If multiple languages are configured, this option will be ignored for Django versions prior to 1.10.`